src/vscode-bicep/src/test/e2e/runTests.ts

```diff
@@ -15,18 +16,28 @@ async function go() {
       vscodeExecutablePath
     );
 
+    // some of our builds run as root in a container, which requires passing
+    // the user data folder path to vs code itself
+    const userDataDir = path.resolve(__dirname, "user-data");
+
+    fs.mkdirSync(userDataDir, { recursive: true });
```

Comment:
From the pipeline logs it seems that this is still not working. I have another idea to workaround this. Maybe we can create another `runTestsInAzureDevOps.ts` file and update the script in `Common.yml` to run the new file `node ./out/test/e2e/runTestsInAzureDevOps.js`

Reply:
Yeah, I saw that. I have a local repro, so I should be able to figure out what happened.

Reply:
Trying a fix that checks for current user name. If `root`, it appends the user data arguments.

Reply:
The fix worked in both pipelines. I'll hold off on merging until you get a chance to take a look.

Reply:
Awesome. One file is definitely better. Looks good to me!